### PR TITLE
feat(policies): document policies

### PR DESCRIPTION
**Pull request by jiparis**
This PR adds a new documentation page, as well as some policy examples to play with.

Refs #122

### CODE REVIEW

File: docs/docs/reference/policies.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+```yaml
+schemaVersion: v1
+materials:
+  - name: sbom
+    type: SBOM_CYCLONEDX_JSON
+  - name: another-sbom
+    type: SBOM_CYCLONEDX_JSON
+policies:
+  materials: # policies applied to materials
+    - ref: sbom-licenses.yaml # (1)
```
Comment (migmartri) [inline docs/docs/reference/policies.md:56]: It's not clear if this is a local file that needs to be somewhere or a name? Did we end up adding file://... to the ref?

**Review by migmartri — APPROVED** [review]
I like it.

The part that seems handwavy and potentially missing is the different kind of references a policy attachment can have.

i.e how to point to a local policy and or a remote policy.

It might make sense to add an example of a remote policy that points to one of the policies you added as well in this patch :)

File: docs/docs/reference/policies.md (hunk 1 above)
Comment (jiparis) [reply docs/docs/reference/policies.md:56]: It's using the `blob.LoadFileOrURL` method from cosign, which defaults to the local filesystem if no `http://`, `https://` or `env://` is provided.

Comment (jiparis) [reply docs/docs/reference/policies.md:56]: I'll make sure it's clear in the docs.

**Pull request merged**